frontend/manifest.json

```diff
@@ -32,5 +32,6 @@
     "48": "assets/cldvn48.png",
     "128": "assets/cldvn128.png"
   },
-  "manifest_version": 2
+  "manifest_version": 2,
+  "content_security_policy": "script-src 'self' https://cdnjs.cloudflare.com"
```

Comment:
Not a fan of loading script from cdnjs for extension for the following reasons:
- no fall back in the case cdnjs is offline
- increased latency compared to loading from local/installed file 

@dshongphuc thought?

Reply:
Yes but it may improve speed in case jQuery has been cached from other websites, and this also help us reduce the size of our extension. I think this is good? @locmai0808

Reply:
I think the best solution is here to not use jQuery at all. Remove jQuery completely. 

Also given that the extension helps filtering out phishing sites, yet loading script from a server (in this case from known CDN), it is still questionable.

Reply:
![image](https://user-images.githubusercontent.com/18728001/117903623-c37b5880-b284-11eb-845a-3311db76a93e.png)

Reply:
@locmai0808 is right, this PR is just a stepping stone for us to completely eliminate the use of jQuery, as I suggest on Trello previously.

Reply:
By the way, please notes that we already use the CDN for bootstrap.min.css (bootstrapcdn, which I think is worse than the CDNJS about speed)